nibe/data/extensions.json

```diff
@@ -49,6 +49,60 @@
       }
     }
   },
+  {
+    "description": "Corrections for fan speed current units",
+    "files": [
+      "f730.json",
+      "f750.json",
+      "f370_f470.json",
+      "f1145_f1245.json",
+      "f1155_f1255.json",
+      "f1345.json",
+      "f1355.json"
+    ],
+    "data": {
+      "41256": {
+        "unit": null,
+        "mappings": {
+          "0": "Normal",
+          "1": "Fan mode 1",
```

Comment:
Does this match how things like uplink name them? Looks a bit weird with the parameter being called fan "speed", but the values being called fan "modes"

Reply:
Hey, no nibe uplink shows the % correctly
![image](https://github.com/user-attachments/assets/ce855b55-6e36-4964-ab09-1d7cbc5967c6)

The configuration on my nibe is Mode 2 == 30%
![image](https://github.com/user-attachments/assets/134c1c05-fcad-4490-a6d1-461dad6bf077)

Homeassistant shows 2%
![image](https://github.com/user-attachments/assets/a408772e-0b0c-4bdd-8c57-55b0a49068ef)

I have a F730 not updated yet to the newest firmware with the new uplink feature. % where shown correctly with the old nibe uplink integration

I don't know if this behaves the same on other series.. just checked where the coil is defined and changed all of them...

After understanding that this is not the real % but the configured active one i would rather use AirFlow
But that has no unit and so displayed as state ;)
https://github.com/yozik04/nibe/pull/163